### PR TITLE
17837- Change of name not showing Jurisdiction for XP

### PR DESCRIPTION
**Pull request by ketaki-deodhar**
*Issue #:* /bcgov/entity#17837

*Description of changes:*

Map to correct EntityType 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-707-0tubf08z.web.app

File: src/list-data/request-action-mapping.ts [hunk 1]
```diff
@@ -81,8 +81,8 @@ export const XproMapping: RequestActionMappingI = {
   ],
   CHG: [
     EntityTypes.A,
-    EntityTypes.XP,
-    EntityTypes.XL,
+    EntityTypes.XLP,
+    EntityTypes.XLL,
```
Comment (severinbeauvais) [inline src/list-data/request-action-mapping.ts:85]: 1. Should we have both XP and XLP (and XL and XLL)?

2. Could you please fix entity-types.ts? XP should be `CorpTypeCd.XPRO_LIM_PARTNR`

Comment (ketaki-deodhar) [reply src/list-data/request-action-mapping.ts:85]: 1. I was a bit confused about keeping it but it makes sense to keep both codes
2. Will do

**Review by severinbeauvais — APPROVED** [review]

**Review by leodube-aot — APPROVED** [review]
LGTM

**Pull request merged**